Views/SelfSizingCollectionView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  SelfSizingCollectionView.swift
+//  Movies
+//
+//  Created by Diana Chizhik on 28/06/2022.
+//
+
+import UIKit
+
+class SelfSizingCollectionView: UICollectionView {
+    override init(frame: CGRect, collectionViewLayout layout: UICollectionViewLayout) {
+        super.init(frame: frame, collectionViewLayout: layout)
+        commonInit()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+
+    private func commonInit() {
+        isScrollEnabled = false
+    }
```

Comment:
Uh I see why you removed the `isScrollEnabled` from the class! Good one.
But you are not using this class there 🤔

Reply:
I found it, I didn't fully understand it, so I left it out to ask about later :)

Reply:
and forgot about disabling scrollView again🤦